env/mock_env.cc

```diff
@@ -378,7 +378,7 @@ class TestMemLogger : public Logger {
       struct timeval now_tv;
       gettimeofday(&now_tv, nullptr);
       const time_t seconds = now_tv.tv_sec;
-      struct tm t{0, 0, 0, 0, 0, 0, 0, 0, 0};
+      struct tm t{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
memset is best given mingw doesn't use gnu c library, and it can be either 9 members
or 11.
struct tm t() is a function declaration. struct tm t is the default constructor but that leaves it uninitialized and was the original problem.